test/e2e-v2/cases/log/extract-timestamp/e2e.yaml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is used to show how to write configuration files and can be used to test.
+
+setup:
```

Comment:
Your new e2e seems not to be added into the GHA control file.

Reply:
So, all the tests are not running.

Reply:
Sorry, I forgot it. I will update it.

Reply:
I am a little confused. Where do you set this log test with the new timestamp style? I can't find it.

Reply:
And I can see this has been tested through UT. I don't think this is so complex that we have to test it through e2e again, right? @kezhenxu94

Reply:
I will remove e2e and add another UT test to check it tomorrow. e2e is too heavy for this case.

Reply:
> I am a little confused. Where do you set this log test with the new timestamp style? I can't find it.

Could you explain this first? I am wondering what I am missing.

Reply:
> > I am a little confused. Where do you set this log test with the new timestamp style? I can't find it.
> 
> Could you explain this first? I am wondering what I am missing.

Here.

![image](https://github.com/apache/skywalking/assets/16346644/13d72b23-7b7b-4460-a4e4-225e06434b58)

Reply:
I saw this, but where the logs of this format generated?

Reply:
> I saw this, but where the logs of this format generated?

The log generated by `provider` defined in docker-compose.yml, it is reported by agent. The format is defined in provider `log4j.xml & logback.xml`:
```
%d{yyyy-MM-dd HH:mm:ss.SSS} [%X{tid}] [%thread] %-5level %logger{36} -%msg%n
```

Reply:
If so, and if you want to keep the e2e, you could add this LAL and verification into `simple` case, it should work too, right?

Reply:
> If so, and if you want to keep the e2e, you could add this LAL and verification into `simple` case, it should work too, right?

I add another UT in `LogTestQueryTest`, it can check the extract result.

Reply:
Could you please show me how to ref the code in comment? I can not find it, I have to pin a picture every time I want show you the code.

Reply:
![image](https://github.com/apache/skywalking/assets/5441976/50ad83a1-c265-49eb-b682-60ee892969e8)

You could get this from the Github pages.

And the permlink is locked for this commit.

Reply:
> ![image](https://user-images.githubusercontent.com/5441976/280167195-50ad83a1-c265-49eb-b682-60ee892969e8.png)
> 
> You could get this from the Github pages.
> 
> And the permlink is locked for this commit.

Thanks, I will try this way next.